src/test/java/com/flagsmith/FlagsmithClientTest.java

```diff
@@ -58,12 +58,23 @@ public void testClient_validateObjectCreation() throws InterruptedException {
         .withConfiguration(
             FlagsmithConfig.newBuilder().withLocalEvaluation(Boolean.TRUE).build()
         )
+        .setApiKey("ser.abcdefg")
         .build();
 
     Thread.sleep(10);
     verify(manager, times(1)).startPolling();
   }
 
+  @Test(groups = "unit")
+  public void testLocalEvaluationRequiresServerKey() throws InterruptedException {
+    Assert.assertThrows(Exception.class, () -> FlagsmithClient.newBuilder()
```

Comment:
Should we verify the exact exception it's throwing here? `RuntimeException.class`?